src/AwsInspector/Command/Ec2/ListCommand.php

```diff
@@ -63,6 +70,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $rows[] = $instance->extractData($mapping);
         }
 
+        $this->sortColumn = $input->getOption('sort');
+        usort($rows, [$this, 'sortByColumn']);
```

Comment:
Just to avoid the usort call if no option is set 

if ($this->sortColumn !== null) {
  usort(...);
}